v63004/u_inref/gtm8909.exp

```diff
@@ -0,0 +1,73 @@
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+set timeout 60
+spawn /usr/local/bin/tcsh -f
+
+expect_after {
+	timeout { timeout_procedure }
+}
+
+proc timeout_procedure { } {
+	puts "timeout occurred"
+	exit -1
+}
+
+puts "# Expect the shell prompt"
+expect -exact ">"
+puts "# Start LKE help facility"
+send -- "\$gtm_dist/lke\r"
+expect -exact "LKE>"
+send -- "help\r"
+puts "# Expect Help Facility Prompt"
```

Comment:
This puts can be removed. It does not add much value particularly when one goes through the reference file where it moves the "help" comand that the user enters into the line after the LKE> prompt so makes it harder to understand the flow.